core/src/main/java/org/testcontainers/containers/FixedHostPortGenericContainer.java

```diff
@@ -12,6 +12,7 @@
  * not available - which could manifest as flaky or unstable tests.</p>
  */
 public class FixedHostPortGenericContainer<SELF extends FixedHostPortGenericContainer<SELF>> extends GenericContainer<SELF> {
+    @Deprecated
```

Comment:
```suggestion
 */
@Deprecated
public class FixedHostPortGenericContainer<SELF extends FixedHostPortGenericContainer<SELF>> extends GenericContainer<SELF> {
```

😂 😂 😂

Reply:
Deprecated without replacement.

Reply:
Why did you do that? It's a very useful class to develop legacy systems where integration tests rely on static configurations and there is no way to override properties dynamically.
It's not an evil because the name is clear and says everything and if you use this class you are aware of risk of port conflicts...